TCAT/Views/SummaryView.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  LiveIndicator.swift
+//  SummaryView.swift
```

Comment:
Also just a general comment: If im not mistaken, this is the view you use to show the summary card with the bus icon and live icon (like the top view in the tableview in your viewcontroller)? Cause for that i would suggest you move to constraints. I imagine you do a lot of pixel point manipulation that's very manual and could be simplified with constraints